otherlibs/site/test/plugin_require_thread.t/run.t

```diff
@@ -0,0 +1,17 @@
+
+  $ dune build ./app.exe @install
+  $ dune exec ./app.exe
+  The library is being used by two plugins finished initialization
+  Fatal error: exception Dune_site_plugins__Plugins.Thread_library_required_by_plugin_but_not_required_by_main_executable
```

Comment:
This test fails with a crash for me locally:

```diff
   Fatal error: exception Dune_site_plugins__Plugins.Thread_library_required_by_plugin_but_not_required_by_main_executable
+  Raised at Dune_site_plugins__Plugins.load_gen in file "otherlibs/site/src/plugins/plugins.ml", line 200, characters 6-89
+  Called from Stdlib__list.iter in file "list.ml", line 110, characters 12-15
+  Called from Dune_site_plugins__Plugins.load_gen in file "otherlibs/site/src/plugins/plugins.ml", line 204, characters 4-36
+  Called from Stdlib__list.iter in file "list.ml", line 110, characters 12-15
+  Called from Dune_site_plugins__Plugins.load_gen in file "otherlibs/site/src/plugins/plugins.ml", line 204, characters 4-36
+  Called from Stdlib__list.iter in file "list.ml",   [2] line 110, characters 12-15
+  Called from Dune__exe__App in file "app.ml", line 2, characters 9-42
   [2]
```

Could you do something about it?

Reply:
Thanks! I'm not sure what "backtrace printing is always activated" means.

Reply:
Maybe something like `export OCAMLRUNPARAM=b` in `~/.bashrc`...

Reply:
@Kakadu I see. That may well be it: my `~/.bashrc` collects settings from various sources, and I wouldn't be surprised if one of them does what you say.